services/configuration/docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust:1.23.0
+
+WORKDIR /usr/src
```

Comment:
Why would you prefer to use a path `/usr/src` ?

Reply:
Well, actually, I don't know. I've used the example from official rust docker image. Is there any drawbacks?

Reply:
Actually this path is used for storing some source files. Linux kernel sources e.g. I'd prefer some thing in `home` directory. E.g. `/home/exonum` Basically it is not important. Just an advice.